shentu-2.2/upgrades/README.md

```diff
@@ -41,3 +41,17 @@ However, we strongly encourage the node operators to stand by during the upgrade
 Make sure you do <b>NOT</b> perform `certik unsafe-reset-all` or delete the data directory, as it will require you to sync from the beginning of shentu-2.2 network. The above steps are sufficient to perform this upgrade.
 
 You can find the release notes and built binaries here: https://github.com/ShentuChain/shentu/releases/tag/v2.4.0.
+
+# Shentu-V260 Upgrade
+
+There is a planned upgrade on shentu-2.2 network at height 10485430, which approximately correspond to `Nov 09, 2022 06:00 UCT`. The upgrade is a simple binary swap <b>at the upgrade height.</b> and rename of home directory from .certik to .shentud
+
+ 1. Upgrade height is reached
+ 2. Stop the running certik daemon
+ 3. Replace the running binary with the v2.6.0 version. The binary is renamed to shentud since v2.6.0.
+ 4. Rename home directory from .certik to .shentud
+ 5. Start the shentud daemon with the new binary
+
```

Comment:
I removed the content on cosmovisor, since we haven't tested the upgrade in this way. Any suggestions/objections?